apps/docs/components/ui/table/TypographyTable.tsx

```diff
@@ -0,0 +1,152 @@
+"use client";
+
+import TypographyPreview from "@/components/preview/TypographyPreview";
+import Code from "@/components/ui/code/Code";
+
+import "./table.css";
+
+interface TypographyTableProps {
+    type: string;
+    data: Record<string, { name: string; value: string }[]>;
+}
+
+const TypographyTable = ({ type, data }: TypographyTableProps) => {
+    type fontProperty = "fontFamily" | "fontSize" | "fontWeight" | "lineHeight";
```

Comment:
fontProperty

types should start with an upercase

also, it should be outside the component